### DISCUSSION TITLE
Moving @database to another volume

### DISCUSSION THREAD

**Discussion by samuelamram** [007revad/Synology_app_mover#156] [Q&A] [answered]
Hi,
Thank you for this excellent script.

Just to confirm: when moving the package `@database` (actually the postgresql database) to another volume using the script, is the database actually **moved** to the new volume, or **copied** and not removed from the current volume?

From what I can see, the database folder remains on the "old" volume even after the move operation successfully completes.

Thank you!

Sam

**Comment by 007revad** [comment, marked as answer]
`@database` is copied to the other volume and DSM's symlink set to point to the other volume.
